.github/workflows/release-snapshot.yml

```
@@ -53,18 +53,28 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
-            const fs = require('fs');
-            const files = await (await glob.create('./packages/*/package.json')).glob();
-            const descriptors = files.map(file => {
-              const { name, version } = JSON.parse(fs.readFileSync(file, 'utf8'));
-              return { name, version };
-            });
-
+            const fs = require("fs");
+            const files = await (await glob.create("./packages/*/package.json")).glob();
+            const descriptors = files
+              .map((file) => {
+                const { name, version } = JSON.parse(fs.readFileSync(file, "utf8"));
+                return { name, version };
+              })
+              .filter(({ version }) => version.includes("-snapshot"));
```

Calling `!snapshot prefix` will trigger a snapshot release using `prefix` instead of `snapshot` for the version name - note that the package will still be published under the `snapshot` tag. Ref: https://github.com/clerkinc/javascript/blob/main/docs/PUBLISH.md#publishing-snapshot-package-versions-snapshot

Checking for `-snapshot` here won't catch versions with custom prefixes. I think we can simply interpolate the output of the `extract-snapshot-name` step here? Eg:

```suggestion
              .filter(({ version }) => version.includes("-${{ steps.extract-snapshot-name.outputs.name }}"));
```

---

Good catch, forgot about that feature!